core/domain/question_domain_test.py

```diff
@@ -32,6 +124,9 @@ def setUp(self):
             feconf.CURRENT_STATES_SCHEMA_VERSION, 'en', 1)
 
     def test_to_and_from_dict(self):
+        """Test to verify verify to_dict and from_dict methods
```

Comment:
Please remove redundant 'verify' from the docstring.

Reply:
OK.